openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/Component/ProfilerSettingsModal.tsx

```diff
@@ -193,7 +198,10 @@ const ProfilerSettingsModal: React.FC<ProfilerSettingsModalProps> = ({
         enablePartition: partitioning.enablePartitioning || false,
       });
 
-      form.setFieldsValue({ ...partitioning });
+      form.setFieldsValue({
+        ...partitioning,
+        partitionValues: partitioning.partitionValues || [''],
+      });
```

Comment:
```suggestion
      form.setFieldValue(
        "partitionValues", partitioning.partitionValues || ['']
      );
```

Reply:
there are multiple fields I am setting here but, from which `partitionValues` is the Form.List so if API don't have data, I am manually setting empty data so one filed is visible by default, without clicking add button

Reply:
You can do it via initialValue then